exercises/hello-workflow/README.md

```diff
@@ -0,0 +1,74 @@
+# Exercise 1: Hello Workflow
+
+During this exercise, you will
+* Review the business logic of the provided Workflow Definition to understand its behavior
```

Comment:
Can you please put a period at the line of each sentence for the objectives and instructions? Thank you! :)